### PR TITLE
My final ACR batch

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/misthioss_fury.txt
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/misthioss_fury.txt:1]: typo in card name

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/misthioss_fury.txt:1]: Corrected (or at least tried).

File: forge-gui/res/cardsfolder/upcoming/senu_keen_eyed_protector.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Senu, Keen-Eyed Protector
+ManaCost:1 W
+Types:Legendary Creature Bird Scout
+PT:2/1
+K:Flying
+K:Vigilance
+A:AB$ GainLife | Cost$ T Exile<1/CARDNAME> | LifeAmount$ 2 | SubAbility$ DBScry | SpellDescription$ You gain 2 life and scry 2.
+SVar:DBScry:DB$ Scry | ScryNum$ 2
+T:Mode$ AttackerUnblocked | ValidCard$ Creature.Legendary+YouCtrl | Execute$ TrigChangeZone | TriggerZones$ Exile | TriggerDescription$ When a legendary creature you control attacks and isn't blocked, if Senu is exiled, put it onto the battlefield attacking.
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/senu_keen_eyed_protector.txt:9]: Typo in trigger (causes crash) needs to be `Execute$ TrigChange`

File: forge-gui/res/cardsfolder/upcoming/phantom_blade.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Phantom Blade
+ManaCost:4 B
+Types:Artifact Equipment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigAttach | TriggerDescription$ When CARDNAME the battlefield, attach it to up to one target creature you control. Tap all creatures target opponent controls.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/phantom_blade.txt:4]: enters

File: forge-gui/res/cardsfolder/upcoming/phantom_blade.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Phantom Blade
+ManaCost:4 B
+Types:Artifact Equipment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigAttach | TriggerDescription$ When CARDNAME enters the battlefield, attach it to up to one target creature you control. Tap all creatures target opponent controls.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/phantom_blade.txt:4]: description wrong

File: forge-gui/res/cardsfolder/upcoming/ratonhnhake_ton.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Ratonhnhaké:ton
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ratonhnhake_ton.txt:1]: does this display fine?

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/ratonhnhake_ton.txt:1]: Circles of Protection also use colon in name field, so it should be fine.

File: forge-gui/res/cardsfolder/upcoming/sigurd_jarl_of_ravensthorpe.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Sigurd, Jarl of Ravensthorpe
+ManaCost:R G W
+Types:Legendary Creature Human Warrior
+PT:3/3
+K:Vigilance
+K:Trample
+K:Lifelink
+A:AB$ AddOrRemoveCounter | Cost$ 1 | Boast$ True | ValidTgts$ Saga.YouCtrl | TgtPrompt$ Select target Saga you control | CounterType$ LORE | CounterNum$ 1
+T:Mode$ CounterAdded | ValidCard$ Saga.YouCtrl | ValidSource$ You | TriggerZones$ Battlefield | CounterType$ LORE | Execute$ TrigPutCounter | TriggerDescription$ Whenever you put a lore counter on a Saga you control, put a +1/+1 counter on up to one other target creature.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl+other | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select another target creature you control | CounterType$ P1P1 | CounterNum$ 1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sigurd_jarl_of_ravensthorpe.txt:10]: bad tgts

File: forge-gui/res/cardsfolder/upcoming/smoke_bomb.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
+Name:Smoke Bomb
+ManaCost:3
+Types:Artifact
+K:Flash
+S:Mode$ Continuous | Affected$ Creature | AddKeyword$ Shroud | Description$ All creatures have shroud. (They can't be the targets of spells or abilities.)
+T:Mode$ Phase | Phase$ Upkeep | TriggerZones$ Battlefield | ValidPlayer$ You | Execute$ TrigSac | TriggerDescription$ At the beginning of your upkeep, sacrifice Smoke Bomb. When you do, target creature you control can't be blocked this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/smoke_bomb.txt:6]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/smoke_bomb.txt [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Name:Smoke Bomb
+ManaCost:3
+Types:Artifact
+K:Flash
+S:Mode$ Continuous | Affected$ Creature | AddKeyword$ Shroud | Description$ All creatures have shroud. (They can't be the targets of spells or abilities.)
+T:Mode$ Phase | Phase$ Upkeep | TriggerZones$ Battlefield | ValidPlayer$ You | Execute$ TrigSac | TriggerDescription$ At the beginning of your upkeep, sacrifice Smoke Bomb. When you do, target creature you control can't be blocked this turn.
+SVar:TrigSac:DB$ Sacrifice | SacValid$ Self | RememberSacrificed$ True | SubAbility$ DBTrigger
+SVar:DBTrigger:DB$ ImmediateTrigger | Execute$ TrigUnblockable | RememberObjects$ RememberedCard | ConditionDefined$ Remembered | ConditionPresent$ Card | SubAbility$ DBCleanup | TriggerDescription$ When you do, target creature you control can't be blocked this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/smoke_bomb.txt:8]: don't need trigger remembering

File: forge-gui/res/cardsfolder/upcoming/ratonhnhake_ton.txt [hunk 8]
```diff
@@ -0,0 +1,13 @@
+Name:Ratonhnhaké:ton
+ManaCost:W U B
+Types:Legendary Creature Human Assassin
+PT:3/3
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Hexproof | IsPresent$ Card.Self+dealtDamagetoAny | PresentCompare$ EQ0 | Description$ As long as CARDNAME hasn't dealt damage yet, it has hexproof and can't be blocked.
+S:Mode$ CantBlockBy | ValidAttacker$ Card.Self | Secondary$ True | IsPresent$ Card.Self+dealtDamagetoAny | PresentCompare$ EQ0 | Description$ As long as CARDNAME hasn't dealt damage yet, it has hexproof and can't be blocked.
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, create a 1/1 black Assassin creature token with menace. When you do, return target Equipment card from your graveyard to the battlefield, then attach it to that token.
+SVar:TrigToken:DB$ Token | TokenScript$ b_1_1_assassin_menace | RememberTokens$ True | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Equipment.YouCtrl | SubAbility$ DBAttach
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ratonhnhake_ton.txt:9]: reflexive trigger missing

**Review by tool4ever — APPROVED** [review]

**Pull request merged**